cwm_worker_operator/domains_config.py

```diff
@@ -448,11 +455,10 @@ def get_cwm_api_volume_config(self, metrics=None, force_update=False, hostname=N
                     metrics.cwm_api_volume_config_error_from_api(worker_id or 'missing', start_time)
             return VolumeConfig(volume_config, self, request_hostname=hostname, is_data_from_cache=False, request_worker_id=worker_id)
         else:
-            if metrics:
-                # success from cache is only possible when we got a worker_id
-                # TODO: add support for cache based on hostname
+            if metrics and worker_id:
                 metrics.cwm_api_volume_config_success_from_cache(worker_id, start_time)
```

Comment:
metrics report should be sent even if worker_id was not provided

code should be like this:

```
volume_config = VolumeConfig(json.loads(val), self, request_hostname=hostname, is_data_from_cache=True, request_worker_id=worker_id)
if metrics and volume_config.id:
  metrics.cwm_api_volume_config_success_from_cache(worker_id, start_time)
return volume_config
```